src/pykeen/sampling/negative_sampler.py

```diff
@@ -25,14 +25,23 @@ def __init__(
         self,
         triples_factory: TriplesFactory,
         num_negs_per_pos: Optional[int] = None,
+        filtered: bool = None,
+        corruption_scheme: Tuple[str] = None,
     ) -> None:
         """Initialize the negative sampler with the given entities.
 
         :param triples_factory: The factory holding the triples to sample from
         :param num_negs_per_pos: Number of negative samples to make per positive triple. Defaults to 1.
+        :param filtered: Whether proposed corrupted triples that are in the training data should be filtered.
+            Defaults to False.
+        :param corruption_scheme: What sides (h, r, t) should be corrupted. Defaults to head and tail (h, t).
         """
         self.triples_factory = triples_factory
         self.num_negs_per_pos = num_negs_per_pos if num_negs_per_pos is not None else 1
+        self.filtered = filtered if filtered is not None else False
```

Comment:
we usually do this trick for other datatypes, but with bools, I think it's just best to put the actual default value in the signature